sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_link_async.py

```diff
@@ -211,7 +213,11 @@ async def _outgoing_flow(self, **kwargs: Any) -> None:
             "echo": kwargs.get("echo"),
             "properties": kwargs.get("properties"),
         }
-        await self._session._outgoing_flow(flow_frame) # pylint: disable=protected-access
+        self._received_drain_response = False
+        # If we aren't still in a drain - for prefetch purposes, we were sending out a flow before receiving a drain
+        if not self._drain_state:
```

Comment:
check here to make sure we aren't sending out Flow frames if we have already sent out a Drain and are waiting on a response